src/OrchardCore.Modules/OrchardCore.Users.TimeZone/ViewModels/UserProfileViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Users.TimeZone.ViewModels
+{
+    public class UserProfileViewModel
+    {
+        public string TimeZone { get; set; }
+        public IList<ITimeZone> TimeZones { get; set; }
```

Comment:
Not really necessary in the view model, it can be accessed directly from the view by injecting `ILocalClock`

Reply:
IClock

Reply:
fixed